sklearn/manifold/tests/test_isomap.py

```diff
@@ -14,66 +14,98 @@
 eigen_solvers = ['auto', 'dense', 'arpack']
 path_methods = ['auto', 'FW', 'D']
 
+N_per_side = 5
+Npts = N_per_side ** 2
```

Comment:
Since we change those line, let's name those variable more explicitly use either lower case or all upper case to respect scikit-learn conventions:

```
N_GRID_TICKS_PER_SIDE = 5
N_GRID_POINTS = N_GRID_TICKS_PER_SIDE ** 2
```